gradle-plugin/plugin/src/main/java/com/yelp/codegen/plugin/GenerateTask.kt

```diff
@@ -83,13 +95,13 @@ open class GenerateTask : DefaultTask() {
             specName ${'\t'} ${specName ?: "[ DEFAULT ] $DEFAULT_NAME"}
             specVers ${'\t'} ${specVersion ?: "[ DEFAULT ] $DEFAULT_VERSION"}
             input ${"\t\t"} $inputFile
-            output ${"\t\t"} ${outputDir ?: "[ DEFAULT ] $defaultOutputDir"}
+            output ${"\t\t"} ${outputDirectory ?: "[ DEFAULT ] $defaultOutputDir"}
             groupId ${'\t'} ${packageName ?: "[ DEFAULT ] default"}
             artifactId ${'\t'} ${packageName ?: "[ DEFAULT ] com.codegen"}
-            features ${'\t'} ${features?.headersToRemove?.joinToString(", ")?.ifEmpty { "[  EMPTY  ]" }}
+            features ${'\t'} ${headersToRemove.joinToString(separator = ",", prefix = "[", postfix = "]")}
```

Comment:
There's a small behaviour change here: if headersToRemove is empty, it will show `[]` instead of `[  EMPTY  ]`